typings/index.d.ts

```diff
@@ -132,7 +132,7 @@ declare module 'discord.js' {
   }
 
   export class CategoryChannel extends GuildChannel {
-    public readonly children: Collection<Snowflake, GuildChannel>;
+    public readonly children: Collection<Snowflake, GuildChannelTypes>;
```

Comment:
wouldn't this collection include `CategoryChannel`s too because `GuildChannelTypes` has it, but, to my knowledge, its not possible to put a category inside a category

Reply:
Technically yes. If we want to make another union type to be more-correct we can, though it isn't a problem if none of the children actually are CategoryChannels. It still enables type-guarding every other channel type that will actually be in there.

Reply:
```suggestion
    public readonly children: Collection<Snowflake, Exclude<GuildChannelTypes, CategoryChannel>>;
```
if you did this, i believe that would exclude category channels from the type